cmd/kops/rollingupdatecluster.go

```diff
@@ -184,12 +196,21 @@ func NewCmdRollingUpdateCluster(f *util.Factory, out io.Writer) *cobra.Command {
 	cmd.Flags().BoolVarP(&options.Interactive, "interactive", "i", options.Interactive, "Prompt to continue after each instance is updated")
 	cmd.Flags().StringSliceVar(&options.InstanceGroups, "instance-group", options.InstanceGroups, "List of instance groups to update (defaults to all if not specified)")
 	cmd.Flags().StringSliceVar(&options.InstanceGroupRoles, "instance-group-roles", options.InstanceGroupRoles, "If specified, only instance groups of the specified role will be updated (e.g. Master,Node,Bastion)")
+	cmd.Flags().BoolVar(&options.Detach, "detach", false, "Detaches the instance from the autoscale group. Useful to let the ASG trigger a new instance.")
```

Comment:
maybe we should mention that this is currently supported only in aws. Like (AWS-only)